src/db/sysdb_ops.c

```diff
@@ -2856,14 +2839,13 @@ int sysdb_store_group(struct sss_domain_info *domain,
         ret = sysdb_store_new_group(domain, name, gid, attrs,
                                     cache_timeout, now);
     } else {
-        ret = sysdb_store_group_attrs(domain, name, gid, attrs,
+        ret = sysdb_store_group_attrs(domain, msg->dn, gid, attrs,
```

Comment:
Wouldn't be the patch smaller / more readable if simply a name part of 'msg->dn' was supplied here, without touching anything else?

Reply:
Thank you for the update.